boards/arm/96b_avenger96/doc/index.rst

```diff
@@ -0,0 +1,276 @@
+.. _96b_avenger96:
+
+96Boards Avenger96
+##################
+
+Overview
+********
+
+96Boards Avenger96 board is based on ST Microelectronics STM32MP157A
+multi-core processor,composed of a dual Cortex®-A7 and a single Cortex®-M4 core.
+Zephyr OS is ported to run on the Cortex®-M4 core.
+
+- Board features:
+
+  - PMIC: STPMIC1A
+  - RAM: 1024 Mbyte @ 533MHz
+  - Storage:
+    - eMMC: v4.51: 8 Gbyte
+    - QSPI: 2Mbyte
+    - EEPROM: 128 byte
+    - microSD Socket: UHS-1 v3.01
+  - Ethernet: 10/100/1000 Mbit/s, IEEE 802.3 Compliant
+  - Wireless:
+
+    - WiFi: 5 GHz & 2.4GHz IEEE 802.11a/b/g/n/ac
+    - Bluetooth: v4.2 (BR/EDR/BLE)
+  - USB:
+    - Host - 2x type A, 2.0 high-speed
+    - OTG: - 1x type micro-AB, 2.0 high-speed
+  - HDMI: WXGA (1366x768)@ 60 fps, HDMI 1.4
+  - Connectors:
+    - 40-Pin Low Speed Header
+    - 60-Pin High Speed Header
+  - LEDs:
+    - 4x Green user LEDs
+    - 1x Blue Bluetooth LED
+    - 1x Yellow WiFi LED
+    - 1x Red power supply LED
+
+.. image:: img/avenger96.png
```

Comment:
Our convention is to name the board image the same as the board ID (so rename it ``96b_avenger96.png`` (and fix the reference to the image here)